HISTORY.md

```diff
@@ -1,4 +1,29 @@
 v3.1.0
+==================
```

Comment:
The ordering is all wrong here.

It goes from 3.1.0, back to 2.2.0, back to 3.1.0 again.